api/types/types.proto

```diff
@@ -492,14 +498,18 @@ message ClusterConfigV3 {
     string SubKind = 2 [ (gogoproto.jsontag) = "sub_kind,omitempty" ];
     // Version is version
     string Version = 3 [ (gogoproto.jsontag) = "version" ];
-    // Metadata is User metadata
+    // Metadata is resource metadata
     Metadata Metadata = 4 [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "metadata" ];
     // Spec is a cluster config V3 spec
     ClusterConfigSpecV3 Spec = 5 [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "spec" ];
 }
 
 // ClusterConfigSpecV3 is the actual data we care about for ClusterConfig.
 message ClusterConfigSpecV3 {
+    reserved 5, 7, 8, 10;
+    reserved "client_idle_timeout", "keep_alive_interval", "keep_alive_count_max",
+        "session_control_timeout";
```

Comment:
We may still need these fields for 7.0.
For example, a 6.0 proxy should be able to read networking options from the old `ClusterConfig` from a 7.0 auth server. Without the proto fields defined, the auth server would stop populating those.